### DISCUSSION TITLE
I cant apply for github student pack

### DISCUSSION THREAD

**Discussion by juanbalella20** [community/community#76900] [GitHub Education]
### Hi everyone,

The problem that I have is that ,when I want to take photo to my certificate of my university, University of Buenos Aires, the page doesn't accept it and I cant find a botton that enable me to upload my certifacte as pdf

**Comment by hackelite01** [comment]
Hey @juanbalella20 , To upload a photo, try clicking on the designated area and selecting the option to upload an image. Alternatively, you can use your phone to directly upload the image. As of now, it's recommended to upload a photo instead of a PDF. If the issue persists, feel free to reach out for further assistance.
